scripts/pre-commit.sh

```diff
@@ -14,14 +14,14 @@ if [ ! "${unstaged_haskell_files}" = "" ]; then
 fi
 
 # Check for unstaged changes in Cabal files
-unstaged_cabal_files="$(git ls-files --exclude-standard --no-deleted --deduplicate --modified '*.cabal')"
-if [ ! "${unstaged_cabal_files}" = "" ]; then
-    echo "Found unstaged Cabal files"
-    echo "${unstaged_cabal_files}"
-    exit 1
-fi
+# unstaged_cabal_files="$(git ls-files --exclude-standard --no-deleted --deduplicate --modified '*.cabal')"
+# if [ ! "${unstaged_cabal_files}" = "" ]; then
+#     echo "Found unstaged Cabal files"
+#     echo "${unstaged_cabal_files}"
+#     exit 1
+# fi
```

Comment:
What's the reason for commenting this out?

Reply:
Ah, already asked :smile:

Reply:
Oops!